index.html

```diff
@@ -563,6 +563,97 @@ <h3>Authorization Capability Request</h3>
     }],
     challenge: '111112b24-63d9-11ea-b99f-4f66f3e4f81a'
   }]
+}
+        </pre>
+      </section>
+    </section>
+    <section>
+      <h3>Logical Operations in Queries</h3>
+
+      <p>
+In Verifiable Presentation Requests, the structuring and retrieval of
+information rely on the use of logical operations. "AND" and "OR" operations
+play crucial roles in defining the path to desired data.
+      </p>
+
+      <section>
+        <h3>Top-Level Queries ("AND" Operation)</h3>
+        
+        <p>
+At the top-most level of our request structure, we assume an implicit "AND"
+operation between different types of queries. Each query is a unique condition
+that needs to be met to fulfill the request. This means, each query listed in
+the top-level array is an independent requirement.
+        </p>
+
+        <p>
+In this example, we have two queries: `APopularQueryType` and
+`AnotherQueryType`. The "AND" operation here implies that both these conditions
+need to be met in order to fulfill the request.
+        </p>
+
+        <pre class="example" title="Multiple credentials requested at once using top-level queries">
+{
+  "query": [
+    // "and"
+    {
+      "type": 'APopularQueryType',
+      // query details ...
+    },
+    // "and"
+    {
+      "type": 'AnotherQueryType',
+      // query details ...
+    }
+  ]
+}
+        </pre>
+      </section>
+
+      <section>
+        <h3>Nested Queries ("OR" Operation)</h3>
+        
+        <p>
+Within specific query types, we can define an "OR" operation. This operation
+indicates that any one of the nested conditions should be met. Even when all
```

Comment:
assuming this is never an XOR ("exclusive or") operation
```suggestion
indicates that one or more of the nested conditions needs to be met. Even when all
```